components/suggest/src/store.rs

```diff
@@ -221,38 +222,48 @@ where
                     .get_records_with_options(&options)?
                     .records;
 
-                self.ingest_records(writer, &records_chunk)?;
+                self.ingest_records(writer, LAST_INGEST_META_UNPARSABLE, &records_chunk)?;
             }
         }
 
+        let ingest_record_types = if let Some(rt) = constraints.allowed_record_types {rt} else {DEFAULT_RECORDS_TYPES.to_vec()};
+        for ingest_record_type in ingest_record_types {
+            self.ingest_records_by_type(ingest_record_type, writer, constraints.max_suggestions)?;
+        }
```

Comment:
To reduce the number of calls to Remote Settings, I can also use the `in_field`. I might change this to do that (will need to update the Remote Settings client though...). I tested that using `in_type` on multiple suggestions type returns all the types inclusive.

Reply:
Oh, that would be wonderful! Yes, please feel free to change the Remote Settings client—we're the only consumer of `GetItemsOptions` now.